### DISCUSSION TITLE
Test API through web.postman.co in GitHub Codespace Forwarded Port

### DISCUSSION THREAD

**Discussion by hashfx** [community/community#30082] [Apps, API and Webhooks]
How can I test Postman API using GitHub Codespace?
Also, ThunderClient or Postman Extension is not working in GitHub Codespace.
Kindly reply with an effective solution ASAP... 🙏🏻🥺

**Comment by jfullstackdev** [comment]
I'm not very sure of your situation but if that is an endpoint and forwarded, then it can be accessed even by the Desktop version of Postman

**Comment by sam-aydev** [comment]
You will need a github token though, follow this link and you will get your answer - https://docs.github.com/en/codespaces/developing-in-a-codespace/forwarding-ports-in-your-codespace
